documentation.css

```diff
@@ -7,38 +7,46 @@
 @import "https://jodd-component-css.netlify.app/component/avatar/avatar.css";
 @import "https://jodd-component-css.netlify.app/component/badge/badge.css";
 @import "https://jodd-component-css.netlify.app/component/button/button.css";
+@import "https://jodd-component-css.netlify.app/component/card/card.css";
 
 
 /* -------header start-------- */
 .header{
-    background-color: var(--PRIMARY-COLOR);
+    background-color: var(--BACKGROUND-COLOR);
     padding: 1rem;
     width: 100vw;
     z-index: 2;
     justify-content: space-between;
+    box-shadow: 1px 1px 7px 3px grey;
```

Comment:
Just one suggestion - see if you reduce the opacity of the box-shadow color based on your taste. that's the first thing I notice since you are using it primarily everywhere. 
```
box-shadow: 1px 1px 7px 3px rgb(128 128 128 / 20%);
```

Reply:
Yes, the one you suggested looks better. Thank you!!